sql/core/pom.xml

```diff
@@ -153,7 +153,7 @@
     <dependency>
       <groupId>com.h2database</groupId>
       <artifactId>h2</artifactId>
-      <version>1.4.195</version>
+      <version>2.0.202</version>
       <scope>test</scope>
```

Comment:
It's a test-only dependency, so we don't really have vulnerability, but it's good to upgrade anyway.

Reply:
Thanks.